### ISSUE TITLE
Major replay regression - object disappearance

### ISSUE THREAD

**Issue by LinearZoetrope** [SCAII/SCAII#117]
So in replays all the objects except the agent are completely disappearing. This appears to be a problem not only in the display, but the model itself. Things are not loading or happening correctly for some reason.

I was sure I messed everything up in replays somehow (command line was running fine, replays weren't). Then I did a `git bisect`:

```
65aa52c (HEAD, refs/bisect/bad) add arrow rendering
02172e4 add octagon rendering
f32ca93 partial work on octagon, circle
ef4b2f5 (refs/bisect/good-ef4b2f5f9fe5b6d4f82f54c66045de88de76f3f5) kite and triangle now supported - new kite logic not yet tested
```

I can't test the octagon and circle partial work commits directly because the webpage won't load, but the issue is certainly happening at `65aa52c` (when you added arrow rendering), and isn't at `ef4b2f5`. I'm going to bed so I'm not sure what changed, but whatever it is broke the replays.

It's really strange to me that this isn't an RTS issue, because it sure seems like one, but I've tested various commits multiple times and I'm almost certain the bug was introduced in one of the commits I listed up there.

To reproduce:

1. Checkout the corresponding commit
2. Run reinstall on the installer
3. Run `record_agent.py` under the demos
4. Load the webpage and the replay (make sure it's the only replay in the folder for obvious reasons)
5. Press the rewind button, or skip anywhere. All objects will disappear except the agent. It will still move (which is odd since that means it's targeting a unit), but no healthbars or anything move either so it at least appears like the RTS is broken.

**Comment by LinearZoetrope** [comment]
Also fun fact: I googled and you can force a cache invalidation in Chrome with Ctrl+F5, worked every time I needed to fix the webpage due to caching weirdness.

**Comment by jedirv** [comment]
fixed with fbd5e3d, just pushed.
